datacenter/ucp/3.0/guides/admin/install/install-on-azure.md

```diff
@@ -0,0 +1,286 @@
+
+- `"primaryAvailabilitySetName": "****",` - The Worker Nodes availability set.
+- `"vnetResourceGroup": "****",` - If your Azure Network objects live in a 
+seperate resource group.
+- `"routeTableName": "****",` - If you have defined multiple Route tables within
+an Azure subnet.
+
+More details on this configuration file can be found 
+[here](https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/providers/azure/azure.go).
+
+
+
+## Considerations for IPAM Configuration
+
+The subnet and the virtual network associated with the primary interface of
+the Azure VMs need to be configured with a large enough address prefix/range. 
+The number of required IP addresses depends on the number of pods running
+on each node and the number of nodes in the cluster.
+
+For example, in a cluster of 256 nodes, to run a maximum of 128 pods
+concurrently on a node, make sure that the address space of the subnet and the
+virtual network can allocate at least 128 * 256 IP addresses, _in addition to_
+initial IP allocations to VM NICs during Azure resource creation.
+
+Accounting for IP addresses that are allocated to NICs during VM bring-up, set
+the address space of the subnet and virtual network to 10.0.0.0/16. This
+ensures that the network can dynamically allocate at least 32768 addresses,
+plus a buffer for initial allocations for primary IP addresses.
+
+> Azure IPAM, UCP, and Kubernetes
+> 
+> The Azure IPAM module queries an Azure virtual machine's metadata to obtain
+> a list of IP addresses that are assigned to the virtual machine's NICs. The
+> IPAM module allocates these IP addresses to Kubernetes pods. You configure the
+> IP addresses as `ipConfigurations` in the NICs associated with a virtual machine
+> or scale set member, so that Azure IPAM can provide them to Kubernetes when
+> requested.
+{: .important}
+
+#### Additional Notes
+
+- The `IP_COUNT` variable defines the subnet size for each node's pod IPs. This subnet size is the same for all hosts.
+-  The Kubernetes `pod-cidr` must match the Azure Vnet of the hosts. 
+
+## Configure IP pools for Azure stand-alone VMs
+
+Follow these steps once the underlying infrastructure has been provisioned.
+
+### Configure multiple IP addresses per VM NIC
+
+Follow the steps below to configure multiple IP addresses per VM NIC.
+
+1.  Initialize a swarm cluster comprising the virtual machines you created
+    earlier. On one of the nodes of the cluster, run:
+
+    ```bash
+    docker swarm init
+    ```
+
+2.  Note the tokens for managers and workers.
+3.  Join two other nodes on the cluster as manager (recommended for HA) by running:
+
+    ```bash
+    docker swarm join --token <manager-token>
+    ```
+
+4.  Join remaining nodes on the cluster as workers: 
+
+    ```bash
+    docker swarm join --token <worker-token>
+    ```
+
+5.  Create a file named "azure_ucp_admin.toml" that contains contents from
+    creating the Service Principal.
+
+    ```
```

Comment:
Folks installing on Azure may not be very familiar with creating and editing files within Linux. Could we add a quick 
```
touch azure_ucp_admin.toml
vi azure_ucp_admin_toml
```

Or similar?

Reply:
Sure, will add this :)